source_glc/glc_InitMod.F90

```diff
@@ -189,7 +190,7 @@ subroutine glc_initialize(EClock)
    paramfile  = 'unknown_paramfile'
 
    if (my_task == master_task) then
-      open (nml_in, file=nml_filename, status='old',iostat=nml_error)
+      open (newunit=nml_in, file=nml_filename, status='old',iostat=nml_error)
```

Comment:
This is a Fortran 2008 feature, but @mvertens feels it is well supported by the major compilers at this point, and it has been exercised heavily in cmeps testing.